Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -13,7 +13,6 @@ public struct Player
     public Player(PlayerIdentity playerIdentity, PlayerManager playerManager, int startAmount)
     {
         this.playerIdentity = playerIdentity;
-        playerManager.chips = startAmount;
```

Comment:
hahaha this was the source of yesterday's bug I assume?

Reply:
Yep >.>